src/lsm/tree_fuzz.zig

```diff
@@ -598,8 +535,8 @@ pub fn main() !void {
         .read_latency_mean = 0 + fuzz.random_int_exponential(random, u64, 20),
         .write_latency_min = 0,
         .write_latency_mean = 0 + fuzz.random_int_exponential(random, u64, 20),
-        .read_fault_probability = random.uintLessThan(u8, 100),
-        .write_fault_probability = random.uintLessThan(u8, 100),
+        .read_fault_probability = 0,
+        .write_fault_probability = 0,
```

Comment:
Imo testing corruption at this layer is more trouble than it's worth -- it is well covered by the VOPR.